_posts/2020-08-07-g2p.md

```diff
@@ -7,10 +7,10 @@ tags: [ tutorial, explanation, tech-showcase ]
 image: assets/images/bonjour-g2p.png
 description: "How to get from 'a' to 'b' using the g2p library"
 featured: false
-hidden: true
+hidden: false
 ---
 
-This blog post describes the background context for a software tool called `g2p` which is shorthand for 'Grapheme-to-Phoneme', but if you don't know those terms, don't worry, this post will explain it. Essentially, `g2p` is a tool for systematically converting between two types of text. Common use cases include converting between a [writing system and the phonetic alphabet](#use-case-1-getting-the-pronunciation-from-a-words-spelling), between two [writing systems for the same language](#use-case-2-a-language-with-multiple-writing-systems), or between a legacy ['font-hacked' writing system and its current Unicode-supported version](#use-case-3-converting-from-legacy-writing-systems).
+This blog post describes the background context for a software tool called `g2p` which is shorthand for 'Grapheme-to-Phoneme', but if you don't know those terms, don't worry, this post will explain it. Essentially, `g2p` is a tool for systematically converting certain characters[^1] into other ones. Common use cases include converting between a [writing system and the phonetic alphabet](#use-case-1-getting-the-pronunciation-from-a-words-spelling), between two [writing systems for the same language](#use-case-2-a-language-with-multiple-writing-systems), or between a legacy ['font-hacked' writing system and its current Unicode-supported version](#use-case-3-converting-from-legacy-writing-systems).
 
 # What you need to know to understand this post
 
```

Comment:
Good point. Because I'm going to be splitting this up into multiple posts, I think I'll just leave it as-is for now and adjust that section for each post.